components/gitpod-db/go/encryption_test.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+// Licensed under the GNU Affero General Public License (AGPL).
+// See License.AGPL.txt in the project root for license information.
+
+package db
+
+import (
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestAES256CBCCipher_Encrypt_Decrypt(t *testing.T) {
```

Comment:
If we care about backwards compatibility we should have two tests to cover the cross-conversion between existing and new implementation.
WDYT? :thinking:

Reply:
Makes sense. Will add.

Reply:
Added.